konduit-serving-build/src/main/java/ai/konduit/serving/build/deployments/RpmDeployment.java

```diff
@@ -0,0 +1,130 @@
+/*
+ *  ******************************************************************************
+ *  * Copyright (c) 2020 Konduit K.K.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Apache License, Version 2.0 which is available at
+ *  * https://www.apache.org/licenses/LICENSE-2.0.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  * License for the specific language governing permissions and limitations
+ *  * under the License.
+ *  *
+ *  * SPDX-License-Identifier: Apache-2.0
+ *  *****************************************************************************
+ */
+package ai.konduit.serving.build.deployments;
+
+import ai.konduit.serving.build.build.GradlePlugin;
+import ai.konduit.serving.build.config.Deployment;
+import ai.konduit.serving.build.config.DeploymentValidation;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.Accessors;
+import org.apache.commons.lang3.StringUtils;
+import org.nd4j.shade.jackson.annotation.JsonProperty;
+
+import java.io.File;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+@Data
+@Accessors(fluent = true)
+@NoArgsConstructor
+public class RpmDeployment implements Deployment, OsSpecific {
+    public static final String DEFAULT_EXE_NAME = "konduit-serving-deployment.rpm";
+    public static final String PROP_OUTPUTDIR = "rpm.outputdir";
+    public static final String PROP_RPMNAME = "rpm.name";
+
+    private String outputDir;
+    private String rpmName;
+    private String version;
+    private Os osName;
```

Comment:
osName field shouldn't be in the configuration - it should be inferred from Target (Like I've added for Arch)